azext_iot/digitaltwins/common.py

```diff
@@ -5,16 +5,36 @@
 # --------------------------------------------------------------------------------------------
 
 """
-shared: Define shared data types(enums)
+shared: Define shared data types(enums) and constant strings.
 
 """
 
 from enum import Enum
 
+# Retry constants
 MAX_ADT_CREATE_RETRIES = 5
 ADT_CREATE_RETRY_AFTER = 60
-
-
+MAX_ADT_DH_CREATE_RETRIES = 20
+
+
+# Data History strings
+DT_IDENTITY_ERROR = "Digital Twins instance does not have System-Assigned Identity enabled. Please enable and try again."
+ERROR_PREFIX = "Unable to"
+FINISHED_CHECK_RESOURCE_LOG_MSG = "Finished checking the {0} resource."
+ADX_ROLE_MSG = "'Database Admin' permission on the Digital Twins instance for the Azure Data Explorer database '{0}'"
+RBAC_ROLE_MSG = "'{0}' role on the Digital Twins instance for the scope '{1}'"
+TRY_ADD_ROLE_LOG_MSG = "Trying to add the {0}."
+PRESENT_ADD_ROLE_LOG_MSG = "The {0} is already present."
+FINISHED_ADD_ROLE_LOG_MSG = "Finished adding the {0}."
```

Comment:
I like adding output format strings here - this will likely save us some headache finding and fixing messages in the future.

Using `Trying to add the {0}.` as an example - what is the expected format of the input? Maybe we could have a comment for the less obvious messages explaining how it should be used.

Reply:
Added some instructions and an example since these strings are supposed to be built up using earlier message strings.